### DISCUSSION TITLE
How can I translate Lumi to other languages?

### DISCUSSION THREAD

**Discussion by ManazAbdullah** [Lumieducation/Lumi#1298] [Ideas]
I have been testing Lumi for a long time. It is a very good work. I am not a software specialist but I want to translate Lumi to an other language. I have translated many open source programs that are beneficial for people.  
Which file can be translate in Lumi. 
Please help me... Thanks Lumi.

**Comment by sr258** [comment]
Hi @ManazAbdullah.
Lumi currently doesn't support localization of its own language strings. It's something on our agenda, but we haven't come round to implementing this yet. I'll come back to you, once this has been implemented.

H5P itself (which is responsible for most the strings you see when using Lumi) is (nearly) fully localizable. If you contribute to the localization of H5P, it will also show up in Lumi over time. You can use https://translate-h5p.tk/ for this.

**Reply by ManazAbdullah** [reply]
Thank you very much sr258.

**Reply by sr258** [reply]
The core component of Lumi is called h5p-server and it already supports localization. I've added a project to translate-h5p.tk (it's my site) which can now be used to translate some parts of Lumi: https://translate-h5p.tk/weblate/projects/lumi-education/

If you read the guide and register at the site, I'll give you editor rights for Turkish and you can start.

**Comment by JPSchellenberg** [comment]
Thank you for your support. Once I got the i18n framework implemented I will let you know how to translate Lumi.  Into which language do you want to translate?

**Comment by JPSchellenberg** [comment]
If you want to, you could also help to translate our documentation at https://github.com/Lumieducation/Lumi-docs/tree/en . Let me know if you are interested in translating the documentation and I will provide you with further information on how to get started.

**Reply by ManazAbdullah** [reply]
Hi JPSchellenberg,
I want to translate Lumi to Turkish. I am using Linux Ubuntu 21.04 Hirsuit Hippo. I have tested Lumi, it is working very good. I specifically looked assets files in https://github.com/Lumieducation/Lumi-docs/tree/en. These are image files of Lumi.
If you tell me that how translate them, I will start. Thanks.

**Reply by ManazAbdullah** [reply]
I have been working about photos and videos more than 40 years. I am a producer and director.

**Comment by sr258** [comment]
Hi @ManazAbdullah,

Lumi can now be fully localized. We use the facilities at https://translate-h5p.tk for this. There's a project named "Lumi Education", which includes all relevant files. We've added an automatic translation into Turkish, but I'm sure it can be improved.

So, to contribute, please register at translate-h5p, send us a message (c@lumi.education) that you've done so and I will give you editor status. You can then change the translation into Turkish directly.
